python/sqlflow_submitter/import_custom_models.py

```diff
@@ -22,5 +22,6 @@ def import_model_def(estimator_name, namespace):
                 try:
                     namespace[model_name_parts[0]] = __import__(
                         model_name_parts[0])
+                    return model_name_parts[0]
```

Comment:
shall we add some log if the estimator_name is not in "a.b" format? Or, we just intend to ignore it.

Reply:
I think just ignore it will be fine.